### PR TITLE
[Doc] [Features] Some cosmetic enhancements

### PR DESCRIPTION
**Pull request by fabschurt**
These are some tiny cosmetic fixes in the `.feature` files.

### CODE REVIEW

File: features/basics/partial_test_doubles.feature [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Feature: Partial test doubles
 
   A _partial test double_ is an extension of a real object in a system that is instrumented with
-  test-double like behaviour in the context of a test. This technique is very common in Ruby
+  test-double-like behaviour in the context of a test. This technique is very common in Ruby
```
Comment (JonRowe) [inline features/basics/partial_test_doubles.feature]: This isn't grammatically correct.

**Comment by JonRowe** [comment]
If you could revert the extra hyphen I'll merge this, thanks :)

File: features/basics/partial_test_doubles.feature (hunk 1 above)
Comment (fabschurt) [inline features/basics/partial_test_doubles.feature]: Woops, sorry, my bad. But in that case, I should also remove the hyphen in [this line](https://github.com/rspec/rspec-mocks/blob/master/features/basics/partial_test_doubles.feature#L22) (I mistakenly thought that this was the right syntax to use), shouldn't I?

Comment (JonRowe) [inline features/basics/partial_test_doubles.feature]: Please :)

**Comment by fabschurt** [comment]
All done. I used `rebase -i` and `push -f` to amend the bad commit.

**Comment by JonRowe** [comment]
Thanks :)

**Comment by fabschurt** [comment]
My pleasure :smile:

**Pull request merged**